src/components/Modals.css

```diff
@@ -0,0 +1,32 @@
+.close-button {
+    position: absolute;
+    right: 13px;
+    top: 13px;
+}
+
+.modal-card {
+    border-radius: 25px;
+    text-align: center;
+}
+
+.decision-button {
+    border: none;
+    font-size: 17px;
+    color: #fffffe;
+    padding: 10px 16px;
+    margin: 15px;
+    border-radius: 8px;
+    cursor: pointer;
+}
+
+.decision-button:hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+button.accept {
+    background-color: #4e5fae;
+}
+
+button.reject {
+    background-color: #ecb8b8;
+}
```

Comment:
Could you change this to Modal.scss

Also we can leverage some Sass tricks to make this cleaner, I would start by nesting everyting in the modal class

```
.modal {
    .modal-card { ... }
    .decision-button { ... }
}

Reply:
I looked to a different approach where I only pass one 'type' prop to the Modal, and, in the constructor, I decide with a switch the description and title. Don't know if there may be issues with it, but thought that always having to pass long props would not be ideal.